msl_expressions/autogenerated/src/Plans/Behaviours/AlignToGoal.cpp

```diff
@@ -215,17 +215,19 @@ namespace alica
         shared_ptr < geometry::CNPoint2D > aim = make_shared < geometry::CNPoint2D > (x, y);
         double samplePoints = 4;
 
-        for (double i = 0.0; i < samplePoints; i += 1.0)
+        for (double i = 0.0; i <= samplePoints; i += 1.0)
```

Comment:
+ Bugfix

Reply:
Is this really intended to run 5 times with 4 samplePoints? Also why is i of type double when it is not further used?